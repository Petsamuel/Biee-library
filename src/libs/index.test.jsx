// eslint-disable-next-line no-unused-vars
import React from "react";
import renderer from "react-test-renderer";
import { describe, expect, test } from "vitest";
import { Test } from "./index";

describe("test", () => {
  test("test component renders correctly", () => {
    const component = renderer.create(<Test />);

    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });

  test("The test prop works", () => {
    const component = renderer.create(<Test name="samuel" />);

    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
