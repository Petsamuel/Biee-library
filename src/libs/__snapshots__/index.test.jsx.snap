// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Buttons > Button component renders correctly 1`] = `
<button
  className=""
>
  <span
    className="p-4 font-[Raleway]"
  >
    Button
  </span>
</button>
`;

exports[`Buttons > The Button prop works 1`] = `
<button
  className="bg-green-500 hover:bg-green-700 p-4 rounded-md hover:shadow-lg text-white hover:text-black"
  onClick={[Function]}
>
  <span
    className="p-4 font-[Raleway]"
  >
    samuel
  </span>
</button>
`;

exports[`button > The button prop works 1`] = `
<button
  className="bg-blue-500 hover:bg-blue-700 p-4 rounded-md hover:shadow-lg  "
>
  <span
    className="p-4 font-[Raleway]"
  >
    samuel
  </span>
</button>
`;

exports[`button > button component renders correctly 1`] = `
<button
  className=""
>
  <span
    className="p-4 font-[Raleway]"
  >
    Button
  </span>
</button>
`;

exports[`test > The test prop works 1`] = `
<h1
  className=""
>
  My name is 
  samuel
</h1>
`;

exports[`test > test component renders correctly 1`] = `
<h1
  className=""
>
  My name is 
</h1>
`;

exports[`test > test component renders correctly 2`] = `
<button
  className=""
>
  <span
    className="p-4 font-[Raleway]"
  >
    Button
  </span>
</button>
`;
